src/genie/libs/parser/iosxe/show_ip.py

```diff
@@ -110,5 +110,250 @@ def cli(self, output = None):
+
+
+# =======================================
+# Schema for:
+#  * 'show ip bgp vpnv4 vrf {vrf} summary'
+# =======================================
+class ShowIpBgpVpnv4VrfUserSummarySchema(MetaParser):
+    """Schema for show ip bgp vpnv4 vrf {vrf} summary."""
+
+    schema = {
+            "rid": str,
+            "local_as": int,
+            "table_version": int,
+            "main_routing_table_version": int,
+            "network_entries": int,
+            "network_entries_memory_usage_bytes": int,
+            "multipath_network_entries": int,
+            "multipath_paths": int,
+            "path_attribute_entries": int,
+            "best_path_attribute_entries": int,
+            "as_path_entries": int,
+            "as_path_entries_memory_usage_bytes": int,
+            "extended_community_entries": int,
+            "extended_community_entries_memory_usage_bytes": int,
+            "route_map_cache_entries": int,
+            "route_map_cache_entries_memory_usage_bytes": int,
+            "filter_list_cache_entries": int,
+            "filter_list_cache_entries_memory_usage_bytes": int,
+            "total_memory_usage_bytes": int,
+            "prefix_memory_allocation_count": int,
+            "prefix_memory_release_count": int,
+            "path_memory_allocation_count": int,
+            "path_memory_release_count": int,
+            "neighbor_ips": {
+                str: {
+                    "version": int,
+                    "as": int,
+                    "messages_received": int,
+                    "messages_sent": int,
+                    "table_version": int,
+                    "in_queue": int,
+                    "out_queue": int,
+                    "up_down": str,
+                    "state": int
+                },
+                str: {
+                    "version": int,
+                    "as": int,
+                    "messages_received": int,
+                    "messages_sent": int,
+                    "table_version": int,
+                    "in_queue": int,
+                    "out_queue": int,
+                    "up_down": str,
+                    "state": int
+                },
+                str: {
+                    "version": int,
+                    "as": int,
+                    "messages_received": int,
+                    "messages_sent": int,
+                    "table_version": int,
+                    "in_queue": int,
+                    "out_queue": int,
+                    "up_down": str,
+                    "state": int
+                }
+            }
+        }
+
+
+# =======================================
+# Parser for:
+#  * 'show ip bgp vpnv4 vrf {vrf} summary'
+# =======================================
+class ShowIpBgpVpnv4VrfUserSummary(ShowIpBgpVpnv4VrfUserSummarySchema):
+    """Parser for show ip bgp vpnv4 vrf {vrf} summary"""
+
+    cli_command = 'show ip bgp vpnv4 vrf {vrf} summary'
+
+    def cli(self, vrf="", output=None):
```

Comment:
```vrf``` should be a mandatory argument, please remove ```=""```:
```
def cli(self, vrf, output=None):

Reply:
Updated!